lib/initial-setup/WP_Auth0_InitialSetup_AdminUser.php

```diff
@@ -20,23 +20,18 @@ public function callback() {
 
 		$current_user = wp_get_current_user();
 
-		$db_connection_name = $this->a0_options->get( "db_connection_name" );
-		$domain = $this->a0_options->get( 'domain' );
-		$jwt = $this->a0_options->get( 'auth0_app_token' );
-
 		$data = array(
+			'client_id' => $this->a0_options->get( 'client_id' ),
 			'email' => $current_user->user_email,
 			'password' => $_POST['admin-password'],
-			'connection' => $db_connection_name,
-			'email_verified' => true
```

Comment:
So it seems there is a change in behaviour here as the user will need to manually verify their email. I am okay with this.